openerp/models.py

```diff
@@ -5918,6 +5933,10 @@ def recompute(self):
                     recs.browse(ids)._write(dict(vals))
             # mark computed fields as done
             map(recs._recompute_done, fs)
+        # OpenUpgrade start, reset blacklists:
```

Comment:
Am I missing something? On an Odoo 9.0 database with only 'base' ínstalled without demo data I have this patch:

```
--- a/openerp/models.py
+++ b/openerp/models.py
@@ -5863,6 +5863,7 @@ class BaseModel(object):
         """ Recompute stored function fields. The fields and records to
             recompute have been determined by method :meth:`modified`.
         """
+        print "Recompute called"
         while self.env.has_todo():
             field, recs = self.env.get_todo()
             # determine the fields to recompute

```

 if I do --update all, this is printed 1854 times.